packaging/docker/Dockerfile

```diff
@@ -1,24 +1,32 @@
 ARG BASE_IMAGE=ubuntu:22.04
+ARG BASE_GOLANG_IMAGE=golang
 ARG GOLANG_VERSION
 
+################################################################################
+# Build stage 0
 # Build the apm-server binary. The golang image version is kept
 # up to date with go.mod by Makefile.
-FROM golang:${GOLANG_VERSION} as builder
+################################################################################
+FROM ${BASE_GOLANG_IMAGE}:${GOLANG_VERSION} as builder
 WORKDIR /src
-COPY go.mod go.sum /src/
+COPY go.mod go.sum .go-version /src/
 COPY internal/glog/go.mod /src/internal/glog/go.mod
 RUN --mount=type=cache,target=/go/pkg/mod go mod download
 COPY Makefile *.mk /src
 COPY cmd /src/cmd
 COPY internal /src/internal
 COPY x-pack /src/x-pack
 COPY .git /src/.git
+COPY script /src/script
 
-ENV GOROOT /usr/local/go
```

Comment:
Unrequired for building the wolfi version and in fact it's not needed for the traditional docker image